contracts/SpokePool.sol

```diff
@@ -672,73 +667,6 @@ abstract contract SpokePool is
             exclusiveRelayer,
             uint32(getCurrentTime()),
             uint32(getCurrentTime()) + fillDeadlineOffset,
-            exclusivityDeadline,
```

Comment:
nit: can we add this back? Just to reduce the diff size? I think we have the bytecode space regardless, no?